### PR TITLE
[Inbound outbound] Add outbound calling to stateful test app

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

Update new sample to include the ability to make outbound calls
image

# Why

Allows the testing of both modalities inside the app

https://skype.visualstudio.com/SPOOL/_workitems/edit/3798941
# How Tested

Validated and build locally

### CODE REVIEW

**Review by edwardlee-msft — APPROVED** [review]

File: samples/CallingStateful/src/App.tsx [hunk 1]
```diff
@@ -85,6 +88,7 @@ function App(): JSX.Element {
       } else if (event.removed.length > 0) {
         if (event.removed[0] === call) {
           setCall(undefined);
+          console.log(call.id, call.callEndReason);
```
Comment (PorterNan) [inline samples/CallingStateful/src/App.tsx:91]: Remove console.log

Comment (edwardlee-msft) [reply samples/CallingStateful/src/App.tsx:91]: i thought this was here on purpose

Comment (dmceachernmsft) [reply samples/CallingStateful/src/App.tsx:91]: Since the app is able to handle multiple calls should we not in the sample include the call end reason?

Comment (edwardlee-msft) [reply samples/CallingStateful/src/App.tsx:91]: Since this is a sample app, I am okay with keeping this console log here for quicker problem solving

**Review by PorterNan — APPROVED** [review]
Chat offline, approved. There is a further work item that we isolate different stateful samples - currently the sample is a huge chunk for developers to read, which there might be a potential that it will grow into another composite

The idea of stateful samples are we could have them short and minimum with no logic coupling, but we can always have multiple samples for different purposes

**Pull request merged**